__init__.py

```diff
@@ -204,12 +204,13 @@ def change_pronouns(self, reminder):
         Arguments:
             reminder (str): reminder text
         """
-        my_regex = r'\b{}\b'.format(self.translate('My'))
-        our_regex = r'\b{}\b'.format(self.translate('Our'))
-        your_word = self.translate('Your')
-        reminder = re.sub(my_regex, your_word, reminder)
-        reminder = re.sub(our_regex, your_word, reminder)
-        return reminder
+        vocabulary = self.translate_namedvalues("Perspective")
+
+        reminder = f" {reminder}"
+        for k, v in vocabulary.items():
+            if f" {k} " in reminder:
```

Comment:
I think the `if` is fine. What's the thinking behind a `while`?

Reply:
If the sentence is "remind me that i take my teeth out ouf my mouth". It would only change one. (Pretty rare, but consistent)

Reply:
By default I think `String.replace()` replaces all instances of the search string:
```
>>> reminder = " remind me that i take my teeth out ouf my mouth "
>>> reminder.replace("my", "our")
' remind me that i take our teeth out ouf our mouth '
```

There is an optional parameter if you want to limit that to _n_ instances:
```

>>> reminder.replace("my", "our", 1)
' remind me that i take our teeth out ouf my mouth '
```
So if I'm not mistaken, in this case we should only need to call that once.

It is quite an interesting sentence too lol

Reply:
Doh, that shouldn't happen. (the mistake)